index.js

```diff
@@ -3,15 +3,19 @@
 const fp = require('fastify-plugin')
 const helmet = require('helmet')
 
-module.exports = fp(function (app, options, next) {
+const fastifyHelmet = function (app, options, next) {
   const middleware = helmet(options)
 
   app.addHook('onRequest', function (req, reply, next) {
     middleware(req.raw, reply.raw, next)
   })
 
   next()
-}, {
+}
+
+fastifyHelmet.contentSecurityPolicy = helmet.contentSecurityPolicy
```

Comment:
Can you set it as `module.exports.contentSecurityPolicy` instead? That'd make Node esm named exports work.

Reply:
Yes, thanks for the suggestion, that's better.